packages/test/src/browser/view/test-result-widget.ts

```diff
@@ -83,6 +91,35 @@ export class TestResultWidget extends BaseWidget {
             } else {
                 this.content.append(this.node.ownerDocument.createTextNode(message.message));
             }
+            if (message.stackTrace) {
+                message.stackTrace.map(frame => this.renderFrame(frame));
+            }
+        });
+    }
+
+    renderFrame(stackFrame: TestMessageStackFrame): void {
+        const frameElement = this.node.ownerDocument.createElement('div');
+        frameElement.append(this.node.ownerDocument.createTextNode(stackFrame.label));
+
+        // Add URI information as clickable links
+        if (stackFrame.uri) {
+
+            const uri = stackFrame.uri;
+            frameElement.append(' from ');
```

Comment:
I18N, also, it would be nice if we could make this look a bit like a stack trace, i.e. similar to
```
Uncaught (in promise) TypeError: Cannot destructure property 'startLineNumber' of 'range' as it is undefined.
    at MonacoOutlineContribution.asRange (monaco-outline-contribution.ts:260:17)
    at MonacoOutlineContribution.getFullRange (monaco-outline-contribution.ts:281:21)
    at MonacoOutlineContribution.parentContains (monaco-outline-contribution.ts:245:62)
    at MonacoOutlineContribution.applySelection (monaco-outline-contribution.ts:227:26)
    at MonacoOutlineContribution.createRoots (monaco-outline-contribution.ts:156:14)
    at MonacoOutlineContribution.updateOutline (monaco-outline-contribution.ts:113:43)
    at monaco-outline-contribution.ts:97:92
    at event.ts:176:42
    at CallbackList.invoke (event.ts:184:26)
    at Emitter.fire (event.ts:327:36)
```

Reply:
The message looks like more like a stack trace now.

![image](https://github.com/user-attachments/assets/01017731-94cf-41a8-b8b9-f7c83d3b3ce1)